src/ResultDumper.hpp

```diff
@@ -28,13 +28,13 @@ class ResultDumper : public Result::Reader::Consumer
 public:
   ResultDumper(FILE* stream);
 
-  virtual void on_header(CacheEntryReader& cache_entry_reader);
-  virtual void on_entry_start(uint32_t entry_number,
+  void on_header(CacheEntryReader& cache_entry_reader) final;
```

Comment:
I'm using `override` in other interface implementations (e.g. in `ZstdCompressor.hpp`). How about using `override` here as well?

Reply:
You mean `override` instead of `final`? That would probably silence the warning too.

There is some (small) debate whether `final` is a good idea or not. I'm not really following. I personally usually set final since it's no effort to change it back to override when needed. In does allow further compiler optimizations, although I have not looked at the surrounding code and don't know if this is the case here / relevant here or not.

I can certainly change that if you prefer `override`.